README.md

```diff
@@ -42,6 +42,19 @@ config.plugins.push(new VueSFCAnalyzerWebpackPlugin({
 
 ```
 
+When you run on production build (production mode on Webpack 4), `ModuleConcatenationPlugin` may avoid to calculate sizes for `<script>`, `<template>`.
```

Comment:
"When you run a production build (production mode on Webpack 4), `ModuleConcatenationPlugin` may not calculate sizes for `<script>`, `<template>`."

Reply:
Thanks a0659e0